### PR TITLE
Dockerfile to build tidb, for #522, close #526

### PR DESCRIPTION
**Pull request by netroby**
You must install docker first, then

docker build --rm -t tidb-server .

after build success, you may see images via

docker images

docker run -d --name tidb-server tidb-server

Signed-off-by: ZhiFeng Hu <hufeng1987@gmail.com>

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

**Comment by netroby** [comment]
Need some document explain how to build and how to use. put doc into readme.md or inside docs/ sub directory?
How do you think about?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM golang
+
+VOLUME /opt
+
+RUN apt-get update && apt-get install -y wget git make ; \
+        cd /opt ; \
+        export PATH=$GOROOT/bin:$GOPATH/bin:$PATH ; \
+        go get -v github.com/pingcap/tidb ; \
```
Comment (siddontang) [inline Dockerfile]: we will use godep to build tidb, so go get -d is enough, no need -v.

**Comment by coocood** [comment]
LGTM

**Pull request merged**